### DISCUSSION TITLE
XZ Exploit and poetry?

### DISCUSSION THREAD

**Discussion by DavidSantacruzR** [python-poetry/poetry#9262] [Q&A] [answered]
I'd like to know, if the recent found exploit somewhat affects poetry or if poetry uses any of the contaminated versions of xz utils 5.6.x and liblzma. Thanks.

**Comment by radoering** [comment, marked as answer]
Poetry itself is a pure Python application and thus does not use xz utils itself. It might be possible that one of its dependencies uses xz utils. However, I am not aware that this is the case.
